internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerRoute.java

```diff
@@ -101,7 +101,7 @@ public enum ControllerRoute {
   STORE("/store", HttpMethod.GET, Collections.singletonList(NAME)), // get all information about that store
   NEW_STORE(
       "/new_store", HttpMethod.POST, Arrays.asList(NAME, KEY_SCHEMA, VALUE_SCHEMA), OWNER, IS_SYSTEM_STORE,
-      ACCESS_PERMISSION
+      ACCESS_PERMISSION, CLUSTER
```

Comment:
Not used?

Reply:
I think the `CLUSTER` arg is used by the code but is missing here. It might even be mandatory

Will take a look and remove if it is unrelated to this change or add it to the mandatory param list

Reply:
It is mandatory in most/all Controller routes. Added it in required configs